qemu/tests/cfg/memory_leak_after_nichotplug.cfg

```diff
@@ -0,0 +1,5 @@
+- memory_leak_after_nichotplug:
+    type = memory_leak_after_nichotplug
+    pci_model = virtio-net-pci
+    enable_msix_vectors = no
```

Comment:
Sorry, one more tiny issue, this line could be removed right?

Reply:
The default value of enable_msix_vectors is yes.
See guest-hw.cfg +37.
Anyway, it doesn't matter whether it's enabled or disabled in this case. So keep it or rm it are both ok I think.

Reply:
@xiywang I see, found the usage in command line generation, let's keep it as you wish.